src/parent-controller.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.parent', ['openfin.window'])
+        .controller('ParentCtrl', ['$scope', 'windowCreationService', function($scope, windowCreationService) {
+            var config = {
+                'autoShow': true,
+                'minWidth': 918,
+                'minHeight': 510,
+                'defaultWidth': 1280,
+                'defaultHeight': 720,
+                'frame': false,
+                'url': 'index.html'
+            };
+
+            windowCreationService.ready(function() {
+                // TODO: Restore correct window(s)
+                windowCreationService.createWindow(config, function(newWindow) {
+                    newWindow.show();
+                });
+
+                $scope.$on('updateFavourites', function(event, data) {
```

Comment:
This and the accompanying `broadcast` seem to be the only time you're leveraging angular on the parent window. More of a debate, but do you think it's needed for it to be wrapped up in angular services, just for this case? It *might* make it seem a little simpler to not wrap up in angular services expose some bindings on the child via one variable (like `fin`), which gets wrapped up in the desktop/childWindow service.